src/Symfony/Component/Config/Definition/EnumNode.php

```diff
@@ -24,13 +24,8 @@ class EnumNode extends ScalarNode
 
     public function __construct($name, NodeInterface $parent = null, array $values = array())
     {
-        $values = array_unique($values);
-        if (count($values) <= 1) {
```

Comment:
I would keep a validation to forbid empty arrays (which does not make sense as it makes the config unusable)